index.html

```diff
@@ -24,6 +24,24 @@ <h1>Connect</h1>
       <p>github: vpostyka@github.com</p>
       <p>linkedIn: vlad</p>
     </section>
+    <section>
+      <h2>Leave message</h2>
+      <form name="leave_message">
+        <label for="usersName">Name:</label>
```

Comment:
To match the lesson formatting, you need <br> elemnats after the labels and textArea.

Reply:
Add more <br> elements.  

 <form name="leave_message">
        <label for="usersName">Name:</label><br>
        <input type="text" name="usersName" required />
        <br />
        <label for="usersEmail">Email:</label><br>
        <input type="email" name="usersEmail" required />
        <br />
        <label for="usersMessage">Message: </label><br>
        <textarea name="usersMessage" required></textarea><br>
        <button type="submit">Submit</button>
      </form>

Reply:
Add more <br> elements to match the formatting.  After each label and after the textarea.